conda/__init__.py

```diff
@@ -42,6 +42,7 @@
     os.environ[str('CONDA_ROOT')] = sys.prefix
 
 CONDA_PACKAGE_ROOT = abspath(dirname(__file__))
+CONDA_SITE_PACKAGES = dirname(CONDA_PACKAGE_ROOT)
```

Comment:
"site-packages" has a significant meaning in Python packaging as the default place where packages are installed in, which makes me wonder if we should try to prevent using it in this case. Somewhere in this PR I saw the name "CONDA_SOURCE_ROOT" which may be better suited and prevents the overlap in meaning?

Reply:
Good call